source/libs/features.tsx

```diff
@@ -169,11 +168,11 @@ const add = async (definition: FeatureDetails) => {
 			return result;
 		};
 		onAjaxedPages(() => run(details));
-	} else if (isPromise(load)) {
+	} else if (load instanceof Promise) {
```

Comment:
That's weird. Because `p-is-promise` has a typeguard in its type definition, so it should work the same... https://github.com/sindresorhus/p-is-promise/blob/959b6f32892c494b57656bdba24e895057c9fbae/index.d.ts#L13